tasks/src/main/resources/logback.xml

```diff
@@ -0,0 +1,34 @@
+<!--
+  ~ Copyright (C) 2023 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
```

Comment:
Nit: copyright year ?

Reply:
As I copied the file, I did not want to change the copyright year.

Reply:
If the file was copied, the copyright year should not be updated.